nemo_curator/modifiers/doc_modifier.py

```diff
@@ -26,3 +26,12 @@ def __init__(self):
     @abstractmethod
     def modify_document(self, text):
         pass
+
+    def get_backend(self) -> str:
+        """
+        The dataframe backend the filter operates on.
+        Can be 'pandas', 'cudf', or 'any'. Defaults to 'pandas'.
+        Returns:
+            str: A string representing the dataframe backend the filter needs as input
+        """
+        return "pandas"
```

Comment:
Same as above on type hint + abstractproperty + require explicit definition in child class rather than abstract class